.env

```diff
@@ -79,8 +79,8 @@ _APP_COMPUTE_RUNTIMES_NETWORK=runtimes
 _APP_EXECUTOR_SECRET=your-secret-key
 _APP_EXECUTOR_HOST=http://exc1/v1
 _APP_FUNCTIONS_RUNTIMES=php-8.0,node-18.0,python-3.9,ruby-3.1
-_APP_SITES_RUNTIMES=static-1,node-22,flutter-3.24
-_APP_SITES_FRAMEWORKS=sveltekit,nextjs,nuxt,astro,remix,static,flutter # TODO: Angular
+_APP_SITES_RUNTIMES=static-1,ssr-22,flutter-3.24
+_APP_SITES_FRAMEWORKS=sveltekit,nextjs,nuxt,astro,remix,static,flutter,other # TODO: Angular
```

Comment:
Do we need both static and other?

Reply:
Good one 🔥 We dont

Reply:
Removed from frameworks, needed in runtimes